discovery-infra/test_infra/tools/assets.py

```diff
@@ -44,6 +50,110 @@ def release_all(self) -> None:
         self._release()
 
 
-class NetworkAssets(Assets):
+class LibvirtNetworkAssets(Assets):
+    """ An assets class that stores values based on the current available
+        resources, in order to allow multiple installations while avoiding
+        conflicts. """
+
     def __init__(self):
         super().__init__(assets_file=consts.TF_NETWORK_POOL_PATH)
+        self._allocated_ips_objects = []
+        self._allocated_bridges = []
+
+    def get(self) -> Munch:
```

Comment:
i still think that this function must run with filelock

Reply:
what will the filelock gives here?

Reply:
it will block any other get action. Just use const filename and it will block same thing that you are trying to block in base test

Reply:
not the same thing, prepare nodes is the part that actually allocates the resources, but I changed the logic so it is not relevant anymore.